docs/content/docs/operators/feature/robustscaler.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "Robust Scaler"
+weight: 1
+type: docs
+aliases:
+- /operators/feature/robustscaler.html
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## Robust Scaler
+
+An Estimator which scales features using statistics that are robust to outliers.
```

Comment:
The description here should stand for the algorithm as a whole, including both `RobustScaler` and `RobustScalerModel`. Thus it might be improper to say "Estimator" here.